models/repo/release.go

```diff
@@ -372,6 +387,34 @@ func GetReleaseAttachments(ctx context.Context, rels ...*Release) (err error) {
 		sortedRels.Rel[currentIndex].Attachments = append(sortedRels.Rel[currentIndex].Attachments, attachment)
 	}
 
+	// Makes URL's predictable
+	for _, release := range rels {
+		// If we have no Repo, we don't need to execute this loop
+		if release.Repo == nil {
+			continue
+		}
+
+		// Check if there are two or more attachments with the same name
+		isDoubled := false
+		foundNames := make(map[string]bool)
+		for _, attachment := range release.Attachments {
+			_, found := foundNames[attachment.Name]
+			if found {
+				isDoubled = true
+				break
+			} else {
+				foundNames[attachment.Name] = true
+			}
+		}
+
+		// If the names unique, use the URL with the Name instead of the UUID
+		if !isDoubled {
```

Comment:
```suggestion
		hasDuplicates := false
		foundNames := make(map[string]bool)
		for _, attachment := range release.Attachments {
			_, found := foundNames[attachment.Name]
			if found {
				hasDuplicates = true
				break
			} else {
				foundNames[attachment.Name] = true
			}
		}

		// If the names unique, use the URL with the Name instead of the UUID
		if !hasDuplicates {
```